dot/peerset/peerset.go

```diff
@@ -348,6 +348,7 @@ func (ps *PeerSet) allocSlots(setIdx int) error {
 		}
 
 		if n.getReputation() < BannedThresholdValue {
+			logger.Warnf("reputation is lower than banned threshold value")
```

Comment:
nit let's add some additional context. Feel free to add more if you feel it's relevant!

```suggestion
			logger.Warnf(
				"reputation %d is lower than banned threshold value %d", 
				n.getReputation(), BannedThresholdValue)
```